src/utils.jl

```diff
@@ -24,9 +24,16 @@ function facedofs(cell, local_face_idx::Int, field_idx::Int)
 end
 
 Ferrite.vertices(cell::Ferrite.Cell{3,3,1}) = cell.nodes
+
 Ferrite.default_interpolation(::Type{Ferrite.Cell{3,3,1}}) = Ferrite.Lagrange{2,Ferrite.RefTetrahedron,1}()
-face_cell(cell::Ferrite.Tetrahedron, i::Int) =  Ferrite.Cell{3,3,1}(Ferrite.faces(cell)[i])
-face_cell(cell::Ferrite.Hexahedron, i::Int) = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[i])
+
+# Note: This extracts the face spanned by the vertices, not the actual face!
+linear_face_cell(cell::Ferrite.Tetrahedron, local_face_idx::Int) =  Ferrite.Cell{3,3,1}(Ferrite.faces(cell)[local_face_idx])
+linear_face_cell(cell::Ferrite.Hexahedron, local_face_idx::Int) = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[local_face_idx])
+linear_face_cell(cell::Ferrite.Cell{3,20,6}, local_face_idx::Int) = Ferrite.Quadrilateral3D(Ferrite.faces(cell)[local_face_idx])
+
+# Obtain the face interpolation on regular geometries.
+getfaceip(ip::Ferrite.Interpolation{dim, shape, order}, local_face_idx::Int) where {dim, shape <: Union{Ferrite.RefTetrahedron, Ferrite.RefCube}, order} = Ferrite.getlowerdim(ip)
```

Comment:
See above - this way we directly support e.g. pyramidal elements.

Reply:
However, note that this is still just a hotfix and this functionality will likely be removed in the future again.

Reply:
ok so in case the `local_face_idx` is the one where the pyramid is a quad you return something special instead of `Ferrite.getlowerdim(ip)` or do I still don't get it ?

Reply:
The interpolation carries the geometry, hence the call to `getlowerdim` becomes ambiguous. I.e. is it `Lagrange{2,RefCube,order}` or `Lagrange{2,RefTetrahedron,order}`.